assemblyline/odm/base.py

```diff
@@ -57,6 +57,8 @@
 MAC_REGEX = r"^(?:(?:[0-9a-f]{2}-){5}[0-9a-f]{2}|(?:[0-9a-f]{2}:){5}[0-9a-f]{2})$"
 URI_PATH = r"(?:[/?#]\S*)"
 FULL_URI = f"^((?:(?:[A-Za-z]*:)?//)?(?:\\S+(?::\\S*)?@)?(?:{IP_REGEX}|{DOMAIN_REGEX})(?::\\d{{2,5}})?){URI_PATH}?$"
+PLATFORM_REGEX = r"^(Windows|Linux)$"
+PROCESSOR_REGEX = r"^x(64|86)$"
```

Comment:
ARM?

Reply:
From what I can find online, ARM is a family of processor architectures that uses 32 or 64-bit. What I'm trying to do here is indicate whether an operating system is 32-bit or 64-bit, so should I call it `System_Bits` instead of `Processor` in order to make it clearer and set the regex to `r"^(32|64)-bit$"`?

Reply:
leave it as is then